Utilities/bootstrap

```diff
@@ -176,6 +176,9 @@ class Target(object):
 
         # Enable the workaround for SR-1457 until it is fixed.
         other_args.extend(["-DEnableWorkaroundForSR1457"])
+
+        if args.libdispatch_source_dir:
+            other_args.extend(["-Xcc -fblocks"])
```

Comment:
I would expect this to be `["-Xcc", "-fblocks"]` and am surprised it actually works, if not (that might mean we have a quoting bug somewhere else).